build.yml

```diff
@@ -269,6 +269,28 @@ docker-targets:
       xorg-x11-fonts-Type1
       zlib
 
+  archlinux:
+    source: docker/Dockerfile.archlinux
+    args:
+      from: archlinux:latest
+    output: pacman
```

Comment:
While [fpm has pacman support](https://github.com/jordansissel/fpm/tree/v1.10.2#things-that-should-work), that's [not supported yet in the build script](https://github.com/wkhtmltopdf/packaging/blob/0.12.6-2/build.yml#L7). You can either use `tar` or add support -- probably [adding an `elsif` around line 163](https://github.com/wkhtmltopdf/packaging/blob/0.12.6-2/build#L163) should be straightforward.